change-feature-layer-renderer/src/main/java/com/esri/arcgisruntime/samples/changefeaturelayerrenderer/MainActivity.java

```diff
@@ -94,21 +94,20 @@ private void createBottomToolbar() {
             @Override
             public boolean onMenuItemClick(MenuItem item) {
                 // Handle action bar item clicks
-                switch (item.getItemId()) {
-                    // handle th
-                    case R.id.action_override_rend:
-                        // check the state of the menu item
-                        if (!overrideActive) {
-                            overrideRenderer();
-                            // change the text to reset
-                            overrideActive = true;
-                            item.setTitle(R.string.action_reset);
-                        } else {
-                            resetRenderer();
-                            // change the text to override
-                            overrideActive = false;
-                            item.setTitle(R.string.action_override_rend);
-                        }
+                int itemId = item.getItemId();
```

Comment:
Can you add the comment line about why this uses if-else, like you did for the others? Worth adding these for when people go to update the code.